docs/docs/gatsby-internals-terminology.md

```diff
@@ -6,8 +6,16 @@ Throughout the Gatsby code, you'll see the below object fields and variables men
 
 ## Page
 
+<Accordion>
```

Comment:
Obfuscating the terminology in collapsed sections on this page doesn't seem super helpful, as users can no longer scan the contents. I'd recommend not using the accordion here.